cloudify_nsx/security/group.py

```diff
@@ -65,6 +65,10 @@ def create(**kwargs):
             raise cfy_exc.NonRecoverableError(
                 "Security group '%s' already exists" % group['name']
             )
+        elif use_existing:
+            raise cfy_exc.NonRecoverableError(
+                "Security group '%s' does not exists" % group['name']
```

Comment:
(and yet "Security group <x> exists" is correct, so don't update that- English is a weird language...)